lib/fp.js

```diff
@@ -124,37 +116,34 @@ const curry = (
   return curryUntil(condition, fn, ...args);
 };
 
-const curryN = (
-  // Curry fn count times, first curry uses args for first currying
-  fn, // function, curried
-  count, // number, of times function should be curried
-  ...args // array, arguments for first currying
-  // Returns: function, curried given times count
-) => {
+// Curry fn count times, first curry uses args for first currying
+//   fn - <Function>, to be curried
+//   count - <number>, of times function should be curried
+//   args - <Array>, arguments for first currying
+// Returns: <Function>, curried given times count
+const curryN = (fn, count, ...args) => {
   let i = -1;
   const condition = () => (i++, i === count);
   return curryUntil(condition, fn, ...args);
 };
 
-const curryTwice = (
-  // Curry function curry with fn
-  fn // function, to be curried
-  // Returns: function, to pass arguments that returns curried fn
-) => curry(curry, fn);
-
-const applyArgs = (
-  // Apply arguments
-  ...args // array, arguments to save in closure
-  // Returns: function, to pass (fn) arguments will be applied
-) => fn => fn(...args);
-
-const either = (
-  // Get first not errored result of fn
-  fn // function, to be called
-  // Returns: result of `fn`
-) => (
-  ...args // arguments to iterate
-) => {
+// Curry function curry with fn
+//   fn - <Function>, to be curried
+// Returns: <Function>, to pass arguments that returns curried fn
+const curryTwice = fn => curry(curry, fn);
+
+// Apply arguments
+//   args - <Array>, arguments to save in closure
+// Returns: <Function>, to be applied saved arguments
+//   args - <Array>, arguments saved in closure
+const applyArgs = (...args) => fn => fn(...args);
+
+// Get first not errored result of fn
+//   fn - <Function>, to be called
+// Returns: <any>, result of `fn(...args)`
```

Comment:
I'd prefer for the nested Result to be indented at the same level as the function.

Reply:
Than what do you think about
```js
// Get first not errored result of fn
//   fn - <Function>, to be called
// Returns: <Function>
//   args - <Array>, arguments to iterate
//   Returns: <any>, result of `fn(...args)`
// Throws: <Error>, if `fn` throws it
```
so it will be obvious which `Returns` belongs to which function?

Reply:
wdyt of
```
// Get first not errored result of fn
//   fn - <Function>, to be called
// Returns: 
//   <Function>
//     args - <Array>, arguments to iterate
//   Returns: <any>, result of `fn(...args)`
// Throws: <Error>, if `fn` throws it
```
OR
```
// Get first not errored result of fn
//   fn - <Function>, to be called
// Returns: <Function>
//     args - <Array>, arguments to iterate
//   Returns: <any>, result of `fn(...args)`
// Throws: <Error>, if `fn` throws it
```
?

Reply:
@lundibundi, I, personally, like the second variant more, but in general I am fine with both of them.

Reply:
As for me, second option looks better.